proto/accessgraph/v1alpha/query.proto

```diff
@@ -29,11 +29,13 @@ service AccessGraphService {
   // GetFile gets a static UI file from the access graph container.
   rpc GetFile(GetFileRequest) returns (GetFileResponse);
 
-  // SendEvent sends an event to the access graph service.
-  rpc SendEvent(SendEventRequest) returns (SendEventResponse);
-
-  // SendResource sends a resource to the access graph service.
-  rpc SendResource(SendResourceRequest) returns (SendResourceResponse);
+  // EventsStream is a stream of commands to the access graph service.
+  // Teleport Auth server creates a stream to the access graph service
+  // and pushes all resources and following events to it.
+  // This stream is used to sync the access graph with the Teleport database state.
+  // Once Teleport finishes syncing the current state, it sends a sync command
+  // to the access graph service and resumes sending events.
+  rpc EventsStream(stream EventsStreamRequest) returns (google.protobuf.Empty);
```

Comment:
Can we not return the proto.empty? If we ever decide to return something, we will need a new endpoint as Empty cannot be changed. With an empty message, you can just add a field when needed.